examples/full-jaffle-shop-demo/dbt/models/subscriptions.yml

```
@@ -220,11 +220,11 @@ models:
               filtered_by_plan:
                 type: count_distinct
                 label: Subscriptions by Plan
-                description: Count of subscriptions for specific plan type (uses plan_type)
+                description: Count of subscriptions for specific plan type (supports multiple values)
                 sql: |
                   CASE 
-                    WHEN ${ld.parameters.plan_type} = 'all' THEN subscription_id
-                    WHEN ${plan.plan_name} = ${ld.parameters.plan_type} THEN subscription_id
```

If you stop using the reference `${plan.plan_name}`, Lightdash won't join the plan table.

---

@ZeRego `${plan.plan_name}` was causing it to silently skip these metrics though. This works because the join is happening in the sql file. 

Should it be in both places?

---

We should investigate why it is failing 🤔 because it should work as mentioned in these docs https://docs.lightdash.com/references/metrics#custom-sql-in-aggregate-metrics

---

It was hidden because of user attributes